open-data/Lesson6-Sharing&Reuse.md

```diff
@@ -0,0 +1,311 @@
+
+* Various flavors of the Creative Commons License [https://creativecommons.org/](https://creativecommons.org/) 
+
+* The British Open Government License- [https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)
+
+* The World Bank Terms of Use - [https://www.worldbank.org/en/about/legal/terms-and-conditions](https://www.worldbank.org/en/about/legal/terms-and-conditions) - among others)
+
+More information on licenses, and copyright can be found at: 
+
+* Digital Curation Center, DCC ([https://www.dcc.ac.uk/guidance/how-guides/license-research-data#x1-13000doc](https://www.dcc.ac.uk/guidance/how-guides/license-research-data#x1-13000doc)) 
+
+* Open Data Commons (https://opendatacommons.org/licenses/),[5](https://creativecommons.org/about/program-areas/open-data/)].
+
+
+In general, it is recommended that the most open license possible is used to enable open data sharing e.g. a Creative Commons CC-0 (waiver of usage restriction), or CC-BY-4 (requires attribution). However, before choosing a license it is a best practice to check with one’s institution to see if there is a standard license that one should use.
+
+Keep in mind that regardless of which license is chosen, reference or use of someone else’s work must be attributed properly, more on that in the next section.  
+
+
+## 6.3 Citations and Acknowledgements
+
+Acknowledgements and citations contribute towards fostering a culture of sharing data without fear of ideas or recognition being stolen. If a researcher can trust that their work will be cited, and used to further the development of science, the idea of making data open is more appealing and mutually beneficial.
+
+Further, data citation aids transparency and allows the original authors and new researchers to easily track how the data is being used to answer different questions, and gain insights from related as well as unrelated fields they wouldn’t ordinarily read about.
+
+Use of standard citation practices are recommended to ensure due credit is given.
+
+## 6.4 Data Access/Data Availability Statements
+
+Sharing data relies on clear statements and methods for accessing the data. This is typically indicated in the Data Availability/Access Statement which includes elements such as; DOI or identifier for the data and citation to the correct references [[9](https://jie.yale.edu/datastatements)], [[10](https://think.f1000research.com/wp-content/uploads/2021/02/F1000Research-Open-Data-How-to-Write-a-Data-Availability-Statement.pdf)] and clear instructions for downloading, using, sharing and citing the data [[11](https://doi.org/10.5281/zenodo.6505587)].  An example of what to include in a good DAS is available in [[10](https://think.f1000research.com/wp-content/uploads/2021/02/F1000Research-Open-Data-How-to-Write-a-Data-Availability-Statement.pdf)] & [[11](https://doi.org/10.5281/zenodo.6505587)].
```

Comment:
```suggestion
Sharing data relies on clear statements and methods for accessing the data. This is typically indicated in the Data Availability/Access Statement which includes elements such as; DOI or identifier for the data and citation to the correct references [[9](https://jie.yale.edu/datastatements)], [[10](https://think.f1000research.com/wp-content/uploads/2021/02/F1000Research-Open-Data-How-to-Write-a-Data-Availability-Statement.pdf)].
It can also include clear instructions for downloading, using, sharing and citing the data [[11](https://doi.org/10.5281/zenodo.6505587)]. 
An example of what to include in a good Availability Statement is available in [[10](https://think.f1000research.com/wp-content/uploads/2021/02/F1000Research-Open-Data-How-to-Write-a-Data-Availability-Statement.pdf)] & [[11](https://doi.org/10.5281/zenodo.6505587)].
```